### PR TITLE
Add PostgreSQL kernel

### PR DESCRIPTION
**Pull request by mgdm**
Hello,
I'm after a bit of feedback on my approach here.

The kernel I've added works, but in order to demonstrate what can be done in the example, it pulls in PostgreSQL itself, and to make that work there's some setup that needs to be done that was easier to do in a script. This means it may not work correctly when run from the test runner. I'm not sure if there's something better I could do there or if I should just rip out the whole thing?

### CODE REVIEW

**Comment by guaraqe** [comment]
This looks nice, thank you!

I understand the need for the PostgreSQL setup, but does it suppose that we already have these tools installed? For example, does this runs also with `nix-shell --pure`? Otherwise, these tools could be added to the shell as a runtime package.

**Comment by guaraqe** [comment]
The `postgres.png` file is missing. Also, when I launch `nix-shell` in the example folder I get the following error:

```
/nix/store/94jxlqmlr777cnw3wa9ycx6651hdpzg5-python3.7-ipython-7.8.0/lib/python3.7/site-packages/IPython/paths.py:68: UserWarning: IPython parent '/homeless-shelter' is not a writable location, using a temp directory.
  " using a temp directory.".format(parent))
error: [Errno 13] Permission denied: '/usr'
```

**Comment by mgdm** [comment]
Whoops, sorry! I've pushed the missing icon now.

I've not seen that particular error but I did do most of it on Nix on OSX. Let me try again on my NixOS machine and get back to you.

**Comment by guaraqe** [comment]
I think the error has something to do with sandboxing, which may not be activated on Mac. I managed to solve one of the errors by doing:

```
preBuild = ''
  export HOME=$TMP
'';
```

But I still see a permission problem:

```
Copying postgres_kernel.egg-info to build/bdist.linux-x86_64/wheel/postgres_kernel-0.2.2-py3.7.egg-info
running install_scripts
error: [Errno 13] Permission denied: '/usr'
```

**Comment by qnikst** [comment]
It looks like that Readme.md is from `ckernel`. It should be something like:

```
  iPostgres = jupyter.kernels.postgresKernel {
    name = "postgres";
  };
```

And I'm not sure but I think it worth mentioning how to set a connection in Readme (even the kernel itself tells it after a failure of the first command).

**Comment by djacu** [comment]
Closing this as it has been re-implemented for the `next` branch in #258.
@mgdm would you mind taking a look and reviewing it? I am not very familiar with PostgreSQL. The kernel does build and I believe it runs fine but I have not setup a database to check it.
